src/components/Button.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+function Button({ clickHandler , toggleDisabled, children, buttonType , nameOfClass }) {
+    return (
+        <div>
+            {nameOfClass ?
+            <button
+                className={ nameOfClass }
+                type={ buttonType }
+                onClick={ clickHandler }
+                disabled={ toggleDisabled }
+            >
+                {children}
+            </button>
+                :
+                <button
+                    className='standardButton'
+                    type={ buttonType }
+                    onClick={ clickHandler }
+                    disabled={ toggleDisabled }
+                >
+                    {children}
+                </button>
+            }
```

Comment:
Ik zie heel veel code duplicatie, terwijl het enige verschil de className is
```suggestion
            <button
                className={ nameOfClass ?? "standardButton" }
                type={ buttonType }
                onClick={ clickHandler }
                disabled={ toggleDisabled }
            >
                {children}
            </button>
```